nss-day-cohort-47/reactive-nutshell-three-https-in-a-pod
# Hp colors

# Description
Added color and minor styling to all pages.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Load up the app and navigate through the pages, styling should be cohesive across all pages.
# Checklist:
